telemetry/telemetry.proto

```diff
@@ -4,6 +4,57 @@ import 'nanopb.proto';
 
 option java_outer_classname="TelemetryProto";
 
-message Telemetry {
-  optional int32 timestamp_ms = 1;
+//Should be sent only after device resets
```

Comment:
Nit: there will need to be a way to request this on-the-fly, in case the telemetry client resets or is started while the telemetry board is already running.